### PR TITLE
Visual indication for deleted schemas/entities (#88)

### PR DESCRIPTION
**Pull request by crazyscientist**
This commit adds indicators to components for showing details and
lists of entities/schemas.

### CODE REVIEW

File: frontend/src/components/EntityListTable.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
       </tr>
       </thead>
       <tbody>
-      <tr v-for="e in entities" :key="e.id">
+      <tr v-for="e in entities" :key="e.id" :class="{'table-light': e.deleted}">
```
Comment (der-gabe) [inline frontend/src/components/EntityListTable.vue:21]: Forgive me, as I'm new to Vue.js, but I don't get this change... we're binding the class attribute (so far, so good), but not to a state or string, but to an to an object, which has `'table-light'` as its only property and the value of it is a boolean, indicating whether the entity `e` is marked as deleted or not... what does this do?

Does it somehow set `class="table-light"` if `e` is deleted?

Comment (crazyscientist) [reply frontend/src/components/EntityListTable.vue:21]: Yes, exactly. That is a [special case](https://vuejs.org/guide/essentials/class-and-style.html) of bindings.

File: frontend/src/components/SchemaList.vue [hunk 2]
```diff
@@ -50,6 +51,12 @@
                 {{ schema.name }}
               </router-link>
             </div>
+            <div>
+              <i v-if="schema.deleted" class="eos-icons" data-bs-toggle="tooltip"
```
Comment (der-gabe) [inline frontend/src/components/SchemaList.vue:55]: We probably don't need to leave an empty `div` here, if the schema is not marked as deleted.
```suggestion
            <div v-if="schema.deleted">
              <i class="eos-icons" data-bs-toggle="tooltip"
```

File: frontend/src/components/EntityListTable.vue (hunk 1 above)
Comment (der-gabe) [reply frontend/src/components/EntityListTable.vue:21]: OK, good to know.

File: frontend/src/components/SchemaList.vue (hunk 2 above)
Comment (crazyscientist) [reply frontend/src/components/SchemaList.vue:55]: Right, currently not. With #69 this might change, but let's go with your suggestion until then.

**Review by der-gabe — APPROVED** [review]
Thanks! :+1:

**Comment by der-gabe** [comment]
Approved, you can merge. Or should I merge, as the Assignee? I don't know what the process in this project is...

**Comment by crazyscientist** [comment]
> you can merge. Or should I merge?

We can discuss the process, but I think its most efficient, when a reviewer performs the merge as part of the review. as I'm already on the page, I can perform the merge :slightly_smiling_face:

**Pull request merged**